README.md

```diff
@@ -1,44 +1,173 @@
+0 | 1 | 0 |1|1|0|0
+0 | 0 | 1 |0|0|1|0
+0 | 1 | 1 |0|0|1|0
+1 | 0 | 0 |1|0|1|1
+1 | 1 | 0 |1|1|0|0
+1 | 0 | 1 |0|0|1|1
+1 | 1 | 1 |0|0|1|1
+
 ### c
 
+A | B | C |NOT B|A OR NOT B| A OR C|(A OR NOT B) AND (A OR C)
+---|---|---|---|---|---|---
+0 | 0 | 0 |1|0|0|0
+0 | 1 | 0 |0|0|0|0
+0 | 0 | 1 |1|0|1|0
+0 | 1 | 1 |0|0|1|0
+1 | 0 | 0 |1|1|1|1
+1 | 1 | 0 |0|0|1|0
+1 | 0 | 1 |1|1|1|1
+1 | 1 | 1 |0|0|1|0
+
 ### d
 
+A | B | C | D |NOT C|NOT A|NOT A AND D|B OR NOT C|NOT (B OR NOT C)|A AND NOT (B OR NOT C)|A AND NOT (B OR NOT C) AND (NOT A AND D)
+---|---|---|---|---|---|---|---|---|---|---|
+0 | 0 | 0 | 0 |1|1|0|1|0|0|0
+0 | 1 | 0 | 0 |1|1|0|1|0|0|0
+0 | 0 | 1 | 0 |0|1|0|0|1|0|0
+0 | 1 | 1 | 0 |0|1|0|1|0|0|0
+1 | 0 | 0 | 0 |1|0|0|1|0|0|0
+1 | 1 | 0 | 0 |1|0|0|1|0|0|0
+1 | 0 | 1 | 0 |0|0|0|0|1|1|0
+1 | 1 | 1 | 0 |0|0|0|1|0|0|0
+0 | 0 | 0 | 1 |1|1|1|1|0|0|0
+0 | 1 | 0 | 1 |1|1|1|1|0|0|0
+0 | 0 | 1 | 1 |0|1|1|0|1|0|0
+0 | 1 | 1 | 1 |0|1|1|1|0|0|0
+1 | 0 | 0 | 1 |1|0|0|1|0|0|0
+1 | 1 | 0 | 1 |1|0|0|1|0|0|0
+1 | 0 | 1 | 1 |0|0|0|0|1|1|0
+1 | 1 | 1 | 1 |0|0|0|1|0|0|0
+
 ## Question 2
 
 ### a
+![alt text](https://github.com/HDonovan96/comp110-worksheet-4/blob/master/Q1A%20Circuit.png)
 
 ### b
+![alt text](https://github.com/HDonovan96/comp110-worksheet-4/blob/master/Q1B%20Circuit.png)
 
 ### c
+![alt text](https://github.com/HDonovan96/comp110-worksheet-4/blob/master/Q1C%20Circuit.png)
 
 ### d
+![alt text](https://github.com/HDonovan96/comp110-worksheet-4/blob/master/Q1D%20Circuit.png)
 
 ## Question 3
 
 ### a
 
+| A | B | A OR B | NOT (A OR B) |
+|---|---|--------|--------------|
+| 0 | 0 | 0      | 1            |
+| 0 | 1 | 1      | 0            |
+| 1 | 0 | 1      | 0            |
+| 1 | 1 | 1      | 0            |
+
+| A | B | NOT A | NOT B | NOT A AND NOT B |
+|---|---|-------|-------|-----------------|
+| 0 | 0 | 1     | 1     | 1               |
+| 0 | 1 | 1     | 0     | 0               |
+| 1 | 0 | 0     | 1     | 0               |
+| 1 | 1 | 0     | 0     | 0               |
+
 ### b
 
+| A | B | A AND B | NOT (A AND B) |
+|---|---|---------|---------------|
+| 0 | 0 | 0       | 1             |
+| 0 | 1 | 0       | 1             |
+| 1 | 0 | 0       | 1             |
+| 1 | 1 | 1       | 0             |
+
+| A | B | NOT A | NOT B | NOT A OR NOT B |
+|---|---|-------|-------|----------------|
+| 0 | 0 | 1     | 1     | 1              |
+| 0 | 1 | 1     | 0     | 1              |
+| 1 | 0 | 0     | 1     | 1              |
+| 1 | 1 | 0     | 0     | 0              |
+
 ### c
 
+| A | B | C | A AND B | A AND C | (A AND B) OR (A AND C) |
+|---|---|---|---------|---------|------------------------|
+| 0 | 0 | 0 | 0       | 0       | 0                      |
+| 0 | 1 | 0 | 0       | 0       | 0                      |
+| 1 | 0 | 0 | 0       | 0       | 0                      |
+| 1 | 1 | 0 | 1       | 0       | 1                      |
+| 0 | 0 | 1 | 0       | 0       | 0                      |
+| 0 | 1 | 1 | 0       | 1       | 1                      |
```

Comment:
This should be 0 -- note that it doesn't match the corresponding row in the second truth table!